ui/pages/routes/routes.component.test.js

```diff
@@ -27,6 +29,7 @@ jest.mock('webextension-polyfill', () => ({
 }));
 
 jest.mock('../../store/actions', () => ({
+  ...jest.requireActual('../../store/actions'),
```

Comment:
Same comments as the other file:
> There must be a reason why the extension team did not want to require the actual selectors and mock only some of them. I see there are no other changes in the file so probably you added it because it is now needed down the tree.
To follow the same patter I would not require the whole file but add only the mock (or mocks) we are missing for the tests.
> Unless an extension team member is OK with requiring the whole selectors file (my rationale here is to follow the existing patterns)

There must be a reason why these actions were individually mocked here instead of requiring them all.

Reply:
see my comment above.